cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -88,6 +88,30 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
             }
           });
         });
+
+        // See https://github.com/bbc/simorgh/pull/4352 for more information.
+        if (service === 'news') {
+          it('plays media when a user clicks play', () => {
+            cy.window().then(win => {
+              const media = getBlockData('video', win.SIMORGH_DATA.pageData);
+              if (media && media.type === 'video') {
+                cy.get(
+                  'div[class^="StyledVideoContainer"] img[class^="StyledImg"]',
+                )
+                  .click()
+                  .should('not.exist')
+                  .then(() => {
+                    cy.get('iframe[class^="StyledIframe"]').then($iframe => {
+                      cy.wrap($iframe.prop('contentWindow'), { timeout: 8000 })
```

Comment:
This seems odd to me, why does the explicit timeout go on `cy.wrap`? Also is the figure of 8s significant?

Reply:
- Timeouts are operate from the 'root' and are applied to chaining methods.
- 8 seconds is double the default - yesterday one of the tests took 5.

Reply:
Just to be clear, are you saying that the 8s is us waiting for the iframe to load? I think what threw me was that it was attached to the `wrap`, which I wouldn't have expected to ever time out.

Reply:
To answer your first question, yes.
To give more information:
`its()` accepts one argument, you cannot apply a timeout to it. Cypress accepts a config object (containing `timeout`) on methods like `get` and `wrap` only. Then any methods (like `its()`), which are chained from that inherit the config.

Reply:
Can we add a comment explaining that this timeout applies to `its` below then and means the test will wait for 8s for the content to start playing?